app/forms/gpo_verify_form.rb

```diff
@@ -25,11 +25,9 @@ def submit
         pending_profile&.deactivate(:in_person_verification_pending)
       elsif fraud_review_checker.fraud_check_failed? && threatmetrix_enabled?
```

Comment:
suggestion: nest the `if` logic here so it's clear that `fraud_review_checker.fraud_check_failed?` is `true` for both current `elsif` clauses:
```
elsif fraud_review_checker.fraud_check_failed?
  if threatmetrix_enabled?
    bump_fraud_review_pending_timestamps
  end
  pending_profile&.activate_after_fraud_review_unnecessary
end
```